Gemfile

```diff
@@ -43,6 +43,7 @@ gem 'oj', '2.10.2' # 2.10.3 causes a 'too deeply nested' error
 gem 'oj_mimic_json'
 gem 'redis'
 gem 'redis-rails', '~> 5.0.2'
+gem 'sprockets', '~>3.7.2'
```

Comment:
Do we need this gem ?

Reply:
Yes, build was failing as it was showing `sprockets` had vulnerability and needed to upgrade to `3.7.2`

Reply:
Ohh Okay, Thanks for clarification 👍